packages/components/src/Banner/Banner.tsx

```diff
@@ -48,6 +48,10 @@ export type BannerProps = {
 };
 
 /**
+ * ```ts
+ * import {Banner} from "@chanzuckerberg/eds-components";
```

Comment:
Can/should this be part of the usage example?

```
* Example usage:
*
* ```tsx
* import {Banner} from "@chanzuckerberg/eds-components";
*
* <Banner
*   onDismiss={handleDismiss}
```

That feels more natural to me, since they go together.

Reply:
I think the intention was to have it at the top of the page to be seen easily. I can also add to example block if you think the block could also use it